src/compiler/diagnosticMessages.json

```diff
@@ -659,7 +659,7 @@
         "category": "Error",
         "code": 1207
     },
-    "All files must be modules when the '--isolatedModules' flag is provided.": {
+    "A global file cannot be compiled using '--isolatedModules'. Ensure your file contains imports, exports, or an 'export {}' statement.": {

```

Comment:
I realized that because this diagnostic is issued on the first statement, we should probably give something more specific.

```
'{0}' cannot be compiled using '--isolatedModules' because it is global. Ensure that it contains an import, export, or an empty 'export {}' statement.
```

Try to make sure that whatever is passed into {0} is the file name, not the full path.

Reply:
@sandersn might have opinions.

Reply:
@DanielRosenwasser Ok, I added proposed changes. Will wait for the feedback from @sandersn